### PR TITLE
install:docker: build ARCH

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
```
export ARCH=$(uname -m) &&  ./install.sh $ARCH
```

works as a one liner

File: GNUmakefile [hunk 1]
```diff
@@ -11,6 +11,11 @@ export TIME
 
 ARCH                                    := $(shell uname -m)
 export ARCH
+#macOS requires aarch64 as ARCH in docker container
```
Comment (RandyMcMillan) [inline GNUmakefile:14]: We can add better Windows detection similar to here.

File: GNUmakefile [hunk 2]
```diff
@@ -11,6 +11,11 @@ export TIME
...
+ifeq ($(ARCH),arm64)
+ARCH                                    := aarch64
+export ARCH
```
Comment (RandyMcMillan) [inline GNUmakefile:17]: We can add better Windows detection similar to here.

**Comment by rsafier** [comment]
NACK, the script works on windows, but made a bunch of invalid containers, none of it works beyond that mkdocs container now

**Comment by rsafier** [comment]
It did work correctly on Ubuntu dev box.

**Pull request closed without merging**